core/api/src/domain/chat-support/index.types.d.ts

```diff
@@ -0,0 +1,8 @@
+type SupportChatId = string & { readonly brand: unique symbol }
+
+type Message = {
+  id: string
+  message: string
+  role: "user" | "assistant"
+  timestamp: number
```

Comment:
must be a Date? (if openai api return a number then we need to translate it in service layer)